e2e/playwright.config.ts

```diff
@@ -48,12 +48,12 @@ export default defineConfig({
     baseURL: process.env.BASE_URL,
     video: 'on-first-retry',
     screenshot: 'only-on-failure',
-    headless: false,
+    headless: true,
     acceptDownloads: true,
     actionTimeout: 20000,
     launchOptions: {
       downloadsPath: 'resources/downloads',
-      args: ['--start-maximized'],
+      args: ['--window-size=1920,1024'],
```

Comment:
FYI this works in CI so this way we can be sure we are always testing with the same resolution